src/Sarif.Multitool/Rules/RuleResources.resx

```diff
@@ -139,17 +139,11 @@ URIs must conform to [RFC 3986](https://tools.ietf.org/html/rfc3986). In additio
 
 If the 'invocation' object specifies both 'startTimeUtc' and 'endTimeUtc', then 'endTimeUtc' must not precede 'startTimeUtc'. To allow for the possibility that the duration of the run is less than the resolution of the string representation of the time, the start time and the end time may be equal.</value>
   </data>
-  <data name="SARIF2001_AuthorHighQualityMessages_Warning_TerminateWithPeriod_Text" xml:space="preserve">
+  <data name="SARIF2001_TerminateMessagesWithPeriod_Warning_Default_Text" xml:space="preserve">
     <value>{0}: In rule '{1}', the message with id '{2}' does not end in a period. Write rule messages as complete sentences.</value>
   </data>
-  <data name="SARIF2001_AuthorHighQualityMessages_FullDescription_Text" xml:space="preserve">
-    <value>Follow authoring practices that make your rule messages readable, understandable, and actionable.
-
-Including "dynamic content" (information that varies among results from the same rule) makes your messages more specific. It avoids the "wall of bugs" phenomenon, where hundreds of occurrences of the same message appear unapproachable.
-
-Placing dynamic content in quotes sets it off from the static text, making it easier to spot. It's especially helpful when the dynamic content is a string that might contain spaces, and most especially when the string might be empty (and so would be invisible if it weren't for the quotes). We recommend single quotes for a less cluttered appearance, even though English usage would require double quotes.
-
-Finally, write in complete sentences and end each sentence with a period. This guidance does not apply to Markdown messages, which might include formatting that makes the punctuation unnecessary.</value>
+  <data name="SARIF2001_TerminateMessagesWithPeriod_FullDescription_Text" xml:space="preserve">
+    <value>Write in complete sentences and end each sentence with a period. This guidance does not apply to Markdown messages, which might include formatting that makes the punctuation unnecessary.</value>
```

Comment:
>Write in complete sentences [](start = 11, length = 27)

we can update them if needed, i used the sections from the original descriptions.

Reply:
That's fine, I'm going to take a pass anyway.

---
In reply to: [447360353](https://github.com/microsoft/sarif-sdk/pull/1945#discussion_r447360353) [](ancestors = 447360353)

Reply:
Actually this is really good. Thanks.

---
In reply to: [447745415](https://github.com/microsoft/sarif-sdk/pull/1945#discussion_r447745415) [](ancestors = 447745415,447360353)